projects/com.oracle.truffle.llvm.parser.base/src/com/oracle/truffle/llvm/parser/base/datalayout/DataLayoutParser.java

```diff
@@ -117,4 +114,13 @@ private static DataLayoutType getDataType(String string) {
         }
     }
 
+    private static Object[] assertNoNullElement(Object[] objects) {
```

Comment:
It's not really an issue for a small helper method but this method is a duplicate of the one in `LLVMParserAsserts`.